<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticDaily</class>
 <widget class="QWidget" name="StatisticDaily">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="vLayoutMain">
   <item>
    <layout class="QHBoxLayout" name="hLayoutInfo"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayoutInput"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vLayoutDataAdder"/>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>380</width>
        <height>247</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="vLayoutResult"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
